Gruntfile.js

```diff
@@ -1,20 +1,60 @@
+/*
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+*/
+
 /* eslint-env node */
 "use strict";
 
 module.exports = function (grunt) {
     grunt.config.init({
+        licenseBanner: grunt.file.read("templates/LICENSE-banner.txt"),
```

Comment:
Unless it causes problems with the plugin, I'd rather see these convenience variables moved under the `usebanner` element, so that the definitions and the references are all together.

Reply:
I tried moving these variables down and the plugin still worked as before.